.circleci/continue_config.yml

```diff
@@ -364,9 +372,7 @@ jobs:
               -GNinja \
               -DENABLE_ALL_WARNINGS=1 \
               -DCMAKE_BUILD_TYPE=Debug \
-              -DPRESTO_ENABLE_PARQUET=ON \
-              -DPRESTO_ENABLE_REMOTE_FUNCTIONS=ON \
-              -DCMAKE_CXX_FLAGS='-Wno-deprecated-declarations -Wno-implicit-int-float-conversion'
```

Comment:
Overriding CMAKE_CXX_FLAGS here was wrong, it seems, because it shadows flags in the CMakeLists.txt file. Also, this doesn't work to resolve a mac-specific issue with arrow_ep mentioned above, so I removed this